test-aether-integration-module/test/__init__.py

```diff
@@ -110,17 +110,21 @@ def entities(client, schemadecorators):  # noqa: F811
 
 
 @pytest.fixture(scope='function')
-def generate_entities(client, mappingset):  # noqa: F811
-    payloads = iter(fixtures.get_submission_payloads())
-    entities = []
-    for i in range(FORMS_TO_SUBMIT):
-        Submission = client.get_model('Submission')
-        submission = Submission(payload=next(payloads), mappingset=mappingset.id)
-        instance = client.submissions.create(data=submission)
-        sleep(2)
-        for entity in client.entities.paginated('list', submission=instance.id):
-            entities.append(entity)
-    return entities
+def generate_entities(realm_client, mappingset):  # noqa: F811
+    def fn(realm):
+        _client = realm_client(realm)
+        payloads = iter(fixtures.get_submission_payloads())
+        entities = []
+        for i in range(FORMS_TO_SUBMIT):
+            Submission = _client.get_model('Submission')
+            submission = Submission(payload=next(payloads), mappingset=mappingset.id)
+            instance = _client.submissions.create(data=submission)
+            sleep(2)
```

Comment:
@shawnsarwar  this does the trick :grimacing:

Reply:
:+1:  Makes sense.